1st 3D Game Tutorial: Setting up the game area update

CyberSkull opened the pull request:
Things updated:
* Grid Snap uses correct shortcut for 4.2
* Screenshot of toolbar with Grid Snap selected.
* macOS shortcut fixes
* small text tweaks


<!--
Please target the `master` branch in priority.
PRs can target other branches (e.g. `3.2`, `3.5`) if the same change was done in `master`, or is not relevant there.
PRs must not target `stable`, as that branch is updated manually.

The type of content accepted into the documentation is explained here:
https://docs.godotengine.org/en/latest/community/contributing/content_guidelines.html
-->

AThousandShips commented:
Please convert the image to the `.webp` format.
you can read more about it in the [Doc image guidelines](https://docs.godotengine.org/en/stable/contributing/documentation/docs_image_guidelines.html#format-conversion).

AThousandShips commented on getting_started/first_3d_game/01.game_setup.rst:
```diff
@@ -21,11 +23,11 @@ In the import popup, enter the full path to the freshly created directory
 contains.
 
-.. image:: img/01.game_setup/02.browse_to_project_folder.webp
+.. image:: img/01.game_setup/02.browse_to_project_folder.png
```
```suggestion
.. image:: img/01.game_setup/02.browse_to_project_folder.webp
```
This is incorrect, the file is `webp`

CyberSkull replied:
Half of the files were PNG, I've converted them all to WEBP.

AThousandShips commented on getting_started/first_3d_game/01.game_setup.rst:
```diff
@@ -21,11 +23,11 @@ In the import popup, enter the full path to the freshly created directory
 Click *Import & Edit* to open the project in the editor.
 
-.. image:: img/01.game_setup/03.import_and_edit.webp
+.. image:: img/01.game_setup/03.import_and_edit.png
```
```suggestion
.. image:: img/01.game_setup/03.import_and_edit.webp
```

CyberSkull replied:
All images converted to WEBP.

AThousandShips commented on getting_started/first_3d_game/01.game_setup.rst:
```diff
@@ -1,3 +1,5 @@
+:article_outdated: True
```
Why this, it was marked not out of date since the 4.2 version, what about the 4.3 version is outdated?

CyberSkull replied:
> Why this, it was marked not out of date since the 4.2 version, what about the 4.3 version is outdated?

I'm not sure where that came from. I'll try a new commit from current source file in a bit.

CyberSkull replied:
It was already on there. I have removed the line.

CyberSkull commented:
I've removed all the PNG files and replaced them with WEBP conversions. The JPEG file has been left in this commit because it is referenced elsewhere.

skyace65 commented on getting_started/first_3d_game/01.game_setup.rst:
```diff
@@ -99,8 +99,10 @@ You should see a wide grey slab that covers the grid and blue and red axes in
 We're going to move the ground down so we can see the floor grid. Select the
-``Ground`` node, hold the :kbd:`Ctrl` key down to turn on grid snapping,
-and click and drag down on the Y axis. It's the green arrow in the move gizmo.
+``Ground`` node, click on the *Use Snap* button (or press the :kbd:`Y` key) to turn on grid snapping,
+and click and drag down on the Y-axis. It's the green arrow in the move gizmo.
```
Ctrl still works for grid snapping so we shouldn't completely remove references to it.
```suggestion
``Ground`` node, holding the :kbd:`Ctrl` key down turns on grid snapping while it's held. You can also
click on the *Use Snap* button (or press the :kbd:`Y` key) to turn on grid snapping.
With snapping enabled using either method, click and drag down on the Y-axis. It's the green arrow in the move gizmo.
```

CyberSkull replied:
Took this into account and rewrote the section.

skyace65 commented:
Thanks. And another reminder, in the future please keep everything to 1 commit, if you don't know how to do that here's a link to the guide in the docs again. https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#modifying-a-pull-request

CyberSkull commented:
> Thanks. And another reminder, in the future please keep everything to 1 commit, if you don't know how to do that here's a link to the guide in the docs again. https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#modifying-a-pull-request

Thanks for the info. I didn't know about the amend feature, it doesn't seem to be supported on the website like it is on the desktop or CLI apps.

skyace65 commented:
> > Thanks. And another reminder, in the future please keep everything to 1 commit, if you don't know how to do that here's a link to the guide in the docs again. https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#modifying-a-pull-request
> 
> Thanks for the info. I didn't know about the amend feature, it doesn't seem to be supported on the website like it is on the desktop or CLI apps.

I would highly recommend learning how to use git on the desktop. That entire page should cover everything you need to know, and if you've got any issues with the instructions please open an issue in this repository.
